packages/trader/src/sass/app/_common/components/contract-type-dialog.scss

```diff
@@ -70,6 +83,8 @@
                 /* postcss-bem-linter: ignore */
                 &__leading-icon {
                     top: 1.2rem;
+                    margin-left: 1rem;
+                    left: 0;
```

Comment:
Should we change this to 
 ```
 margin-inline-start: 1rem;
 inset-inline-start: 0;
```

Reply:
After checking this scss is not needed. Will remove from the code 🙏